tests/ip_config_test.py

```diff
@@ -84,6 +84,18 @@ def test_add_del_interface_valid_ipv4(self):
         assert result.exit_code != 0
         assert ('Eth36.10', '32.11.10.1/24') not in db.cfgdb.get_table('VLAN_SUB_INTERFACE')
 
+        # config int ip add Ethernet0.10 10.21.20.1/24 as secondary
+        result = runner.invoke(config.config.commands["interface"].commands["ip"].commands["add"], ["Vlan100", "10.11.20.1/24", "--secondary"], obj=obj)
+        assert result.exit_code == 0
+        assert ('Vlan100', '10.11.20.1/24') in db.cfgdb.get_table('VLAN_INTERFACE')
+        assert db.cfgdb.get_table('VLAN_INTERFACE')[('Vlan100', '10.11.20.1/24')]['secondary'] == "true"
```

Comment:
nit: add a test where 'secondary' is false for non-secondary cli call.

Reply:
Added the test case for this scenario.